e2e-test/java/io/cdap/plugin/slt/locators/SLTLocators.java

```diff
@@ -0,0 +1,141 @@
+
+    @FindBy(how = How.XPATH, using = "//input[@data-cy='jco.client.sysnr']")
+    public static WebElement jcoClientSysnr;
+
+    @FindBy(how = How.XPATH, using = "//input[@data-cy='jco.client.user']")
+    public static WebElement jcoClientUser;
+
+    @FindBy(how = How.XPATH, using = "//input[@data-cy='jco.client.passwd']")
+    public static WebElement jcoClientPasswd;
+
+    @FindBy(how = How.XPATH, using = "//*[contains(text(),'Next')]")
+    public static WebElement next;
+
+    @FindBy(how = How.XPATH, using = "//input[@data-cy='project']")
+    public static WebElement project;
+
+    @FindBy(how = How.XPATH, using = "//input[@data-cy='datasetName']")
+    public static WebElement datasetName;
+
+    @FindBy(how = How.XPATH, using = "//span[contains(text(),'Configure target')]")
+    public static WebElement configure;
+
+    @FindBy(how = How.XPATH, using = "//span[contains(text(),'Configure advanced properties')]")
+    public static WebElement configureAdvance;
+
+    @FindBy(how = How.XPATH, using = "//span[contains(text(),'Review assessment')]")
+    public static WebElement review;
+
+    @FindBy(how = How.XPATH, using = "//span[contains(text(),'View summary')]")
+    public static WebElement viewSummary;
+
+    @FindBy(how = How.XPATH, using = "//span[contains(text(),'Deploy Replication Job')]")
+    public static WebElement deployPipeline;
+
+    @FindBy(how = How.XPATH, using = "//*[contains(text(), 'Logs')]")
+    public static WebElement logs;
+
+    @FindBy(how = How.XPATH, using = "(//*[contains(text(), 'View')])[1]")
+    public static WebElement advancedLogs;
+
+    @FindBy(how = How.XPATH, using = "//*[contains(text(), 'Running')]")
+    public static WebElement running;
+
+    @FindBy(how = How.XPATH, using = "//*[contains(text(),'Polling next set of files using prefix')]")
+    public static WebElement replication;
+
+    @FindBy(how = How.XPATH, using = "//*[contains(@class, 'icon-stop')]")
+    public static WebElement stop;
+
+    @FindBy(how = How.XPATH, using = "//div[contains(text(),'ERROR')]")
+    public static WebElement error;
+
+    @FindBy(how = How.XPATH, using = "(//div[contains(text(),'ERROR')])[3]")
+    public static WebElement error3;
+
+    @FindBy(how = How.XPATH, using = "//*[contains(text(),'Stopped')]")
+    public static WebElement stopped;
+
+    @FindBy(how = How.XPATH, using = "//*[@class='text-danger']/span")
+    public static WebElement rowError;
+
+    @FindBy(how = How.XPATH, using = "//*[@data-cy='switch-replicateExistingData']/div[2]")
+    public static WebElement replicateExistingData;
+
+    @FindBy(how = How.XPATH, using = "//*[@data-cy='switch-suspendSltJob']/div[2]")
+    public static WebElement suspendSltJob;
+
+    public static By start = By.xpath("//*[contains(@class, 'icon-play ')]");
+
+    public static By replicationStatus = By.xpath("//*[contains(text(),'replication status: Process: ')]");
+
+    public static WebElement table (String tableName) {
```

Comment:
Update the locator name: locateTable(String tableName). 

Use the same naming convention for all the methods in PO class: locateXYZ()